yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
As we don't run npm install/yarn install on this project, should we add this file to `.gitignore` as well?

Reply:
Actually, we cannot as the action would fail when setting up node 😅  but I replace the `yarn.lock` with `package-lock.json` to align with `npm` we use in the workflow

![Screen Shot 2565-04-07 at 11 09 39](https://user-images.githubusercontent.com/1772999/162119067-33cd6f5e-c4f5-42f3-9108-6c9d0d5cedb9.png)

p.s. will update this again once we have a conclusion on [this package manager RFC](https://github.com/nimblehq/react-templates/issues/66)